mix.exs

```diff
@@ -56,7 +56,7 @@ defmodule EctoSQL.MixProject do
       {:telemetry, "~> 0.4.0 or ~> 1.0"},
 
       # Drivers
-      {:db_connection, "~> 2.2"},
+      {:db_connection, "~> 2.5 or ~> 2.4.1"},
```

Comment:
Would it be better to just say `>= 2.4.1`?

Reply:
I think that in this specific case being more restrictive is a win. The behavior is closer to the ideal `"~> X.Y"`

Reply:
Ah, fair enough. I sat and thought about it for a bit and it would cause those who are stuck on an older version to have to override the underlying dep if it strays too far from what is implemented here.